Pod/Classes/NSManagedObject+VOKManagedObjectAdditions.m

```diff
@@ -96,6 +99,21 @@ + (instancetype)vok_addWithDictionary:(NSDictionary *)inputDict forManagedObject
     }
 }
 
++ (void)vok_addWithArrayInBackground:(NSArray *)inputArray completion:(VOKManagedObjectReturnBlock)completion
+{
+    [VOKCoreDataManager importArrayInBackground:inputArray
+                                       forClass:[self class]
+                                     completion:^(NSArray *arrayOfManagedObjectIDs) {
+                                         NSMutableArray *returnArray = [NSMutableArray array];
+                                         NSManagedObjectContext *moc = [[VOKCoreDataManager sharedInstance] managedObjectContext];
+                                         for (NSManagedObjectID *objectID in arrayOfManagedObjectIDs) {
+                                             [returnArray addObject:[moc objectWithID:objectID]];
+                                         }
+                                         completion([returnArray copy]);
```

Comment:
Hmm... actually, I read the article again, and I'd take away two things:
- If you're going to enumerate a collection and you don't trust where it came from, enumerate a copy instead.
- If you're returning a collection that's, on its face, immutable, but you're actually returning a mutable collection that you are (or someone else is) still holding a reference to (and thus can mutate; e.g., a `@property`), return a copy.